src/store/actionTypes.js

```diff
@@ -3,10 +3,13 @@ export const START_RETURN__REQUEST = "START_RETURN:REQUEST"
 export const START_RETURN__SUCCESS = "START_RETURN:SUCCESS"
 export const START_RETURN__FAILURE = "START_RETURN:FAILURE"
 
+export const RETURN_EXCHANGE_DETAILS = "RETURN_EXCHANGE_DETAILS"
```

Comment:
This is a good action to include.

It's helpful when actions have verbs in their name, so when we are viewing it in redux we understand what is happening, for what reason.

To that end, a minor suggested change:

```suggestion
export const UPDATE_RETURN_EXCHANGE_DETAILS = "UPDATE_RETURN_EXCHANGE_DETAILS"
```